### DISCUSSION TITLE
Tarantool logging

### DISCUSSION THREAD

**Discussion by jackcarver-a** [tarantool/tarantool#6851] [Q&A] [answered]
Hi!

What is configure Tarantool logging?

**to file:**
box.cfg{log = 'taran.log'}

**error when starting tarantool cartridge:**
my_app.my-app-1 | 2022-02-10 08:35:23.517 [24] main load_cfg.lua:527 E> ER_CFG: Incorrect value for option 'log': can't be set dynamically

**to syslog:**
box.cfg{log='syslog:server=vector.example.local:605'}

**error when starting tarantool cartridge:**

my_app.my-app-1 | 2022-02-10 08:40:08.872 [24] main load_cfg.lua:527 E> ER_CFG: Incorrect value for option 'log': can't be set dynamically

**Comment by filonenko-mikhail** [comment, marked as answer]
Hello!

## Developing

For local developing use file `instances.yml` in project directory.
There is global section for all tarantool instances :
```
<appname>:
  log: "taran.log"
```

Or use section just for current instance:
```
<appname>.<instance-name>:
  log: "taran-node.log"
```

## Production
For production use file `/etc/tarantool/conf.d/instances.yml`.
There is global section for all tarantool instances :
```
<appname>:
  log: "taran.log"
```

Or use section just for current instance:
```
<appname>.<instance-name>:
  log: "taran-node.log"
```

**Reply by jackcarver-a** [reply]
Hello! Thanks!
